src/tools/copyset_check_core.h

```diff
@@ -105,9 +108,11 @@ const char kThreeCopiesInconsistent[] = "Three copies inconsistent";
 
 class CopysetCheckCore {
  public:
+    explicit CopysetCheckCore(std::shared_ptr<MDSClient> mdsClient) :
```

Comment:
This overloaded constructor is duplicated, you can set the default value of csClient to nullptr in the blow constructor.

Reply:
> This overloaded constructor is duplicated, you can set the default value of csClient to nullptr in the blow constructor.

Change  csClient_ to csClient as a local var for concurrently send rpc, but unit test need to set mock csClient, so add this constructor.

Reply:
> This overloaded constructor is duplicated, you can set the default value of csClient to nullptr in the blow constructor.

done